### ISSUE TITLE
Crash when generating chunks with roosters disabled

### ISSUE THREAD

**Issue by birb-ebooks** [GenDeathrow/Hatchery#115]
I had noticed a long while ago that worlds tended to crash when I had Hatchery and the Chickens mod installed. At first I thought this was because of the new chickens added by the Chickens mod, since the crash happened during generation of the new world (or new chunks in an old world). However, I got the crash again when only using Hatchery.

Here's the crash log from my main heavily modded installation (it mentions Bionmes O' Plenty only because BoP uses the vanilla function to generate animals and stuff, the mod doesn't actually have anything to do with this crash): https://pastebin.com/S4sAmbRG

And here's the crash log from a world with ONLY Hatchery installed: https://pastebin.com/YbbQbbJj

Basically, from what I can tell, the game crashes when it tries to spawn new chunks whenever you have the "Rooster Spawn Chance Probability" config line set to 0. I'm using the recent hotfix currently, but this problem plagued me on the full release version as well.

If you need any more details, let me know!

**Comment by GenDeathrow** [comment]
That's because you cant set a number to 0 with a weight as it will cause it to crash, I'll make sure the next version doesn't allow you to do that. if you want to use a mod that allows you to control other mobs.. like https://minecraft.curseforge.com/projects/in-control

**Comment by birb-ebooks** [comment]
Huh, didn't know that. Thanks for the quick response!
